sql/user-account-management.md

```diff
@@ -93,3 +93,13 @@ Or:
 ```sql
 ALTER USER 'jeffrey'@'localhost' IDENTIFIED BY 'mypass';
 ```
+
+## Flush privileges 
+
+If you modified the grant tables directly, please run the following command to apply changes immediately:
+
+```sql
+FLUSH PRIVILEGES;
+```
+
+See [Privilege Management](privilege.md) for details.
```

Comment:
```suggestion
For details, see [Privilege Management](../sql/privilege.md).
```

Reply:
Is `../sql/` a mistake? Both documents are in sql, so ../sql is the same directory.

Reply:
@lastzero No, `../sql/` is not a mistake. Your link path is correct on GitHub, but on the PingCAP website it will be a broken link. 
Our link rule: Any relative path that links to another file must start from the level of the root directory.
We updated the link path rule for SEO. Thanks for your cooperation :)

Reply:
Interesting. You use pandoc, right? In my own projects, MkDocs automatically creates the right links in HTML as needed.

Reply:
We do use pandoc to convert HTML to PDF, but it has nothing to do with our links in the documents.
We have our own link rule because of the complex hierarchical relationship in our website.